lib/models/repositoryState.js

```diff
@@ -1,6 +1,7 @@
 var _ = require('lodash');
 var Immutable = require('immutable');
 
+var Normalize = require('../utils/normalize');
```

Comment:
Hmm yeah, but it's like every other Utils, they are all uppercased. Same as Immutable just above etc.